Dockerfile

```diff
@@ -1,48 +1,43 @@
-# pull the c-bindings
-FROM rust AS bindings
-RUN cargo install cargo-c
-
-
 # build the gstreamer ndi plugin
-FROM rust AS builder
-COPY --from=bindings /usr/local/cargo/bin /usr/local/cargo/bin
+FROM ubuntu:24.10 AS builder
 
 ENV CARGO_REGISTRIES_CRATES_IO_PROTOCOL="sparse"
 
 RUN apt-get update && apt-get -y install \
+    rustc \
+    cargo \
+    curl \
     build-essential \
     libgstreamer1.0-dev \
     libgstreamer-plugins-base1.0-dev
 
-# GST_PLUGINS_COMMIT=main to use the latest commit
+# ENV GST_PLUGINS_COMMIT=main to use the latest commit
```

Comment:
i think this should be `ARG` instead of `ENV` because it's a build time variable, not runtime

https://docs.docker.com/build/building/variables/#arg-usage-example